### PR TITLE
https://issues.redhat.com/browse/ACM-6589--building gitOps guide and …

### PR DESCRIPTION
**Pull request by swopebe**
…update to push pull from development

### CODE REVIEW

File: gitops/gitops_push_pull.adoc [hunk 1]
```diff
@@ -58,32 +63,49 @@ See the following prerequisites to use the Argo CD _Pull_ model:
...
+[#arch-push]
+=== Architecture Push model
+
+- With Push model, {ocp-short} GitOps applies resources _directly_ from a centralized hub cluster to the managed clusters. 
+
+- An Argo CD application that is running on the hub cluster communicates with the GitHub repository and deploys the manifests directly to the managed clusters. 
 
 - Push model implementation only contains the Argo CD application on the hub cluster, which has credentials for managed clusters. The Argo CD application on the hub cluster can deploy the applications to the managed clusters.
 
+- *Important:* With a large number of managed clusters that require resource application, consider potential strain on the {ocp-short} GitOps controller memory and CPU usage. To optimize resource management, refer to {LINK NEEDED}.
```
Comment (mikeshng) [inline gitops/gitops_push_pull.adoc]: This is the latest link I can see to configure memory usage: https://docs.openshift.com/container-platform/4.13/cicd/gitops/configuring-resource-quota.html

File: gitops/gitops_push_pull.adoc [hunk 2]
```diff
@@ -58,32 +63,49 @@ See the following prerequisites to use the Argo CD _Pull_ model:
...
+
 - By default, the Push model is used to deploy the application unless you add the `apps.open-cluster-management.io/ocm-managed-cluster` and `apps.open-cluster-management.io/pull-to-ocm-managed-cluster` annotations to the template section of the `ApplicationSet`.
 
-[#pull-arch]
-== Pull model architecture
+[#arch-pull]
+=== Architecture Pull model
```
Comment (mikeshng) [inline gitops/gitops_push_pull.adoc]:
```suggestion
=== Architecture Pull model (Technology Preview)
```
Can we say this again just to highlight the fact that it is tech preview when user is comparing the two options?

File: gitops/gitops_push_pull.adoc [hunk 3]
```diff
@@ -58,32 +63,49 @@ See the following prerequisites to use the Argo CD _Pull_ model:
...
+
+- Pull model provides scalability relief compared to the push model by reducing stress on the controller in the hub cluster, but with more requests and status reporting required.
```
Comment (mikeshng) [inline gitops/gitops_push_pull.adoc]: Can we add the word "may" in this sentence? "Pull model may provide...."

**Review by mikeshng — COMMENTED** [review]
Just some comments. This looks really good. Thank you!

File: gitops/gitops_push_pull.adoc (hunk 3 above)
Comment (swopebe) [reply gitops/gitops_push_pull.adoc]: we would say "can"

File: gitops/gitops_push_pull.adoc (hunk 1 above)
Comment (swopebe) [reply gitops/gitops_push_pull.adoc]: Ok, we have to use the portal link, so I have to find that in the official doc and link to it.

**Review by mikeshng — APPROVED** [review]
/approve

/lgtm

**Pull request merged**